mvc/src/main/java/nextstep/mvc/controller/tobe/RequestMappingHandlerMapping.java

```diff
@@ -0,0 +1,40 @@
+package nextstep.mvc.controller.tobe;
+
+import jakarta.servlet.http.HttpServletRequest;
+import java.util.HashMap;
+import java.util.Map;
+import nextstep.mvc.HandlerMapping;
+import nextstep.web.support.RequestMethod;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class RequestMappingHandlerMapping implements HandlerMapping {
+
+    private static final Logger log = LoggerFactory.getLogger(RequestMappingHandlerMapping.class);
+
+    private final Object[] basePackage;
+    private final Map<HandlerKey, HandlerExecution> handlerMappings;
+
+    public RequestMappingHandlerMapping(final Object... basePackage) {
+        this.basePackage = basePackage;
+        this.handlerMappings = new HashMap<>();
+    }
+
+    @Override
+    public void initialize() {
+        handlerMappings.putAll(new ControllerScanner(basePackage).scan());
+
+        log.info("Initialized AnnotationHandlerMapping!");
+        handlerMappings.entrySet()
+                .forEach(entry -> log.info("Path : {}, Controller : {}", entry.getKey(), entry.getValue()));
```

Comment:
```suggestion
        handlerMappings.forEach((key, value) -> log.info("Path : {}, Controller : {}", key, value));
```

Map에서 제공하는 forEach()를 사용해서 entrySet()을 줄일 수 있을 것 같아요!

Reply:
오오.. 감사합니다 바니!
바로 반영했습니다!! ㅎㅎ